src/js/jsx/sections/layers/LayerGroup.jsx

```diff
@@ -68,7 +72,25 @@ define(function (require, exports, module) {
             /**
              * @type {Boolean}
              */
-            disabled: React.PropTypes.bool
+            disabled: React.PropTypes.bool,
+
+            /**
+             * True if the LayerGroup should render collapsed layers. This is controlled by the 
+             * root LayerGroup.
+             * 
+             * @type {Boolean}
+             */
+            renderCollapsedLayers: React.PropTypes.bool
+        },
+        
+        componentDidMount: function () {
+            if (!this.props.parentLayer) {
+                this.renderCollapsedLayers = true;
+
+                window.setTimeout(function () {
+                    this.forceUpdate.bind(this, null);
```

Comment:
Yes that's on purpose. At the beginning, we skip rendering collapsed layers for faster loading, then we wait for 2 seconds to perform a `foreUpdate` to render those collapsed layers. I added the explanation at https://github.com/adobe-photoshop/spaces-design/pull/3503/files#diff-690c204937d25a539891aa9f5a9313fcR43, but you are still confused by this, so maybe I should explain more in here as well?